AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5126] SG2 | Correct pagination link color and add padding

## Ticket(s)

**Jira Ticket**
- [EWL-5126: SG2 | Correct pagination link color and add padding](https://issues.ama-assn.org/browse/EWL-5126)

## Description
This update ensures that when viewing the pagination molecule, there is adequate padding around element and the active link color is purple.

## To Test

- Pull `bugfix/EWL-5126-correct-pagination-color-padding` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Molecules > Pagination](http://localhost:3000/?p=molecules-pagination).
- Confirm there is padding around entire element.

## Visual Regressions

Updated reference images for the event page backstop tests have been included in this PR.


## Relevant Screenshots/GIFs
### Updated pagination molecule
![image](https://user-images.githubusercontent.com/4438120/40450883-9265a702-5ea2-11e8-91b2-ba35e53cd4df.png)


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
